app/src/ui/lib/text-box.tsx

```diff
@@ -177,6 +177,14 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
     )
   }
 
+  private onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    const eventHandler = this.props.onKeyDown
+
+    if (eventHandler != null) {
```

Comment:
Can we skip the intermediate variable here? Doesn't seem to be giving us much over repeating it in the if-statement and the method call